components/calendar/calendar.component.html

```diff
@@ -1,7 +1,7 @@
-<span *ngIf="isShow">
+<span *ngIf="hasBeenOpened" [hidden]="isHidden">
```

Comment:
Why have you renamed isShow ?

Reply:
To make it match the binding and avoid using `!`, it looks better I think

Reply:
In this lib when it's a boolean, the naming convention is **isBlabla**
Your variable does not respect it